src/main/java/org/jabref/logic/journals/Abbreviation.java

```diff
@@ -4,16 +4,22 @@
 
 import javafx.beans.property.SimpleStringProperty;
 
-public class Abbreviation implements Comparable<Abbreviation> {
+import org.apache.commons.lang3.StringUtils;
 
-    private static final String SPLITTER = ";"; // elements after SPLITTER are not used at the moment
+public class Abbreviation implements Comparable<Abbreviation> {
 
     private final SimpleStringProperty name = new SimpleStringProperty("");
     private final SimpleStringProperty abbreviation = new SimpleStringProperty("");
+    private final SimpleStringProperty shortestUniqueAbbreviation = new SimpleStringProperty("");
 
     public Abbreviation(String name, String abbreviation) {
+        this(name, abbreviation, StringUtils.EMPTY);
```

Comment:
oh, this comes form apache. Maybe jsut ust `""` even if it is not that readable.